test/core/binary.wast

```diff
@@ -129,9 +129,9 @@
   "\00asm" "\01\00\00\00"
   "\04\04\01"                          ;; Table section with 1 entry
   "\70\00\00"                          ;; no max, minimum 0, funcref
-  "\09\07\01"                          ;; Element section with 1 entry
-  "\80\00"                             ;; Table index 0, encoded with 2 bytes
-  "\41\00\0b\00"                       ;; (i32.const 0) with no elements
```

Comment:
I'm having a little trouble integrating this in wabt.     Why was an extra byte added to the end here?    What does it represent.   The `\b` terminates the init expt and the a single zero represents the num of elements, no?

Reply:
Note that this is segment format `02` now, which requires an elem kind. That is the first 0 byte after the init expr.

Reply:
I see.  Sorry I think I missed that change.  

Is the documentation out-to-date, or maybe I'm missing something?   From the overview.md and from published spec it looks like the what follows the init_expr is just a list of `func_index`:
https://github.com/WebAssembly/bulk-memory-operations/blob/master/proposals/bulk-memory-operations/Overview.md#element-segments
https://webassembly.github.io/bulk-memory-operations/core/binary/modules.html#binary-elemsec